### PR TITLE
Improve index based commands II

### PR DESCRIPTION
**Pull request by khooroko**
Commands improved: repaid, ban, unban, select, nearby

Select and nearby commands will highlight the next person in the appropriate list and start from the top of the list if called at the end

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/RepaidCommandTest.java [hunk 1]
```diff
@@ -80,6 +80,31 @@ public void execute_invalidIndexFilteredList_throwsCommandException() {
...
+
+    @Test
+    public void execute_noIndexNoSelection_failure() throws Exception {
+        RepaidCommand repaidCommand = prepareCommand();
+
+        assertCommandFailure(repaidCommand, model, Messages.MESSAGE_NO_PERSON_SELECTED);
+    }
+
+    //@@author jaivigneshvenugopal
+
```
Comment (jaivigneshvenugopal) [inline src/test/java/seedu/address/logic/commands/RepaidCommandTest.java]: extra line here

**Review by jaivigneshvenugopal — APPROVED** [review]
Nicely done!

**Review by lawwman — APPROVED** [review]
Good enhancement to the index based commands. Approved.

**Review by jelneo — APPROVED** [review]
LGTM

**Pull request merged**